changelog.d/issue-1232

```diff
@@ -0,0 +1,9 @@
+synopsis: Exported addQueryParam
+packages: servant
+prs: #1785
+issues: #1232
+significance: minor
```

Comment:
@fgaz remind me, is `minor` a valid value for the `significance` field?

Reply:
No, we only have `significance: significant` or omitted

Reply:
@tchoutri btw the docs are here: https://codeberg.org/fgaz/changelog-d